guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

```diff
@@ -27,30 +27,9 @@ Failure to retain these files during an upgrade causes the upgrade to fail.
 If these files have been deleted, they must be restored from a backup in order for the upgrade to proceed.
 ====
 
-ifdef::satellite[]
-.Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts* and select the Content Hosts that you want to upgrade.
-. From the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *{ProjectName} Tools {ProjectVersionPrevious}* checkbox.
-. From the *Select Action* list, select *Override to Disabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *Red Hat {project-client-name}* checkbox.
-. From the *Select Action* list, select *Override to Enabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Packages*.
-. In the *Package* search field, enter one of the following options depending on your configuration:
-+
-* If your deployment uses `katello-agent` and *goferd*, enter `katello-agent`.
-* If your deployment does not use `katello-agent` and *goferd*, enter `katello-host-tools`.
-+
-. Until https://bugzilla.redhat.com/show_bug.cgi?id=1649764[BZ#1649764] is resolved, from the *Update* list, you must select *via remote execution*.
-This is required because if you update the package using the Katello agent, the package update disrupts the communication between the client and {Project} or {SmartProxyServer}, which causes the update to fail.
-For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
-endif::[]
-
 [[cli-upgrading_content_hosts]]
 .CLI Procedure
 . Log into the client system.
-ifdef::satellite[]
```

Comment:
This means the following now gets rendered for Katello/Orcharino, but it doesn't apply there. Remove all the content between the `ifdef` and `endif` please